queries.sql

```diff
@@ -15,3 +15,86 @@ SELECT * from animals WHERE neutered = TRUE;
 SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* TRANSACTIONS */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+
+/* update species */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
```

Comment:
- [ ] Before committing you can simply check if the change was made
`SELECT species from animals;`
COMMIT
`SELECT species from animals;` To check if change persists after commit